requirements_iati.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
+-e git+https://github.com/pwyf/bdd-tester.git#egg=bdd-tester
```

Comment:
@Bjwebb this change is now in its own commit ae819f8d531f592702086d7684fce16c23072fd4